config/grafana/dashboards/load-testing/HTTP failures.json

```diff
@@ -8,19 +8,29 @@
         "hide": true,
         "iconColor": "rgba(0, 211, 255, 1)",
         "name": "Annotations & Alerts",
+        "target": {
+          "limit": 100,
+          "matchAny": false,
+          "tags": [],
+          "type": "dashboard"
+        },
         "type": "dashboard"
       }
     ]
   },
   "editable": true,
-  "gnetId": null,
+  "fiscalYearStartMonth": 0,
   "graphTooltip": 0,
-  "id": 5,
-  "iteration": 1637829287491,
+  "id": 3,
+  "iteration": 1642579601387,
   "links": [],
+  "liveNow": false,
   "panels": [
     {
-      "datasource": "K6-InfluxDB-InfluxQL",
+      "datasource": {
+        "type": "influxdb",
+        "uid": "P4D1D31A5B69203FF"
```

Comment:
please note ,that this id also was added in the datasource provisioning file

Reply:
Great, so only changing the dashboard file would have actually broken it.